test/E2ETests/E2EApps/E2EApp/E2EApp.csproj

```diff
@@ -27,15 +27,12 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <CopyToPublishDirectory>Never</CopyToPublishDirectory>
     </None>
-    <None Update="proxies.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
     <PackageReference Condition="$(TestBuild) == 'true'" Include="Microsoft.Azure.Functions.Worker" Version="1.0.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.3.0" />
```

Comment:
Should we switch the Worker.Sdk package reference to a ProjectReference so that it picks up the latest bits when running these tests?

Reply:
The powershell script that runs this locally will replace the line with a new, local package with the latest bits. The pipeline also does this as well. Not sure why we opted for that instead of a ProjectReference.

Reply:
If I recall -- the msbuild targets files that the SDK uses in the package don't get picked up with project refs. That's why we need this overly complicated build with the `devpack.ps1` file, etc.